workos/__about__.py

```diff
@@ -12,7 +12,7 @@
 
 __package_url__ = "https://github.com/workos-inc/workos-python"
 
-__version__ = "5.33.0"
+__version__ = "5.34.1"
```

Comment:
nit: just for posterity would you mind changing the title of this PR to v5.34.1?

Reply:
oh wait I can, lol. I'll do it and approve, haha